knitout-to-kcode.js

```diff
@@ -156,28 +157,34 @@ function merge_types(a,b) {
 		if (a !== TYPE_XFER_FOUR_PASS && a !== TYPE_XFER_TWO_PASS) return a;
 		else return null;
 	}
-	
+
 	//types that only define one bed get merged:
 	if (a === TYPE_KNIT_x) {
-		if      (b === TYPE_x_KNIT || b === TYPE_KNIT_KNIT) return TYPE_KNIT_KNIT;
+		if (b === TYPE_x_KNIT || b === TYPE_KNIT_KNIT) return TYPE_KNIT_KNIT;
 		else if (b === TYPE_x_TUCK || b === TYPE_KNIT_TUCK) return TYPE_KNIT_TUCK;
+		else if (b === TYPE_SPLIT_TO_BACK) return TYPE_KNITFRONT_SPLIT2BACK; //* //?
 	} else if (a === TYPE_x_KNIT) {
-		if      (b === TYPE_KNIT_x || b === TYPE_KNIT_KNIT) return TYPE_KNIT_KNIT;
+		if (b === TYPE_KNIT_x || b === TYPE_KNIT_KNIT) return TYPE_KNIT_KNIT;
 		else if (b === TYPE_TUCK_x || b === TYPE_TUCK_KNIT) return TYPE_TUCK_KNIT;
+		else if (b === TYPE_SPLIT_TO_FRONT) return TYPE_KNITBACK_SPLIT2FRONT; //* //?
```

Comment:
Is other token needed? Would `TYPE_KNIT` + `TYPE_SPLIT_TO_FRONT` => `TYPE_SPLIT_TO_FRONT` work fine?

Reply:
yeah, I'll change it to this.